### DISCUSSION TITLE
When i get to 08 step in dpgen . dpdata made a mistake.

### DISCUSSION THREAD

**Discussion by qinmi-sudo** [deepmodeling/dpgen#548] [Q&A] [answered]
![e36bc7bf28a59439e091b9df4372fb0](https://user-images.githubusercontent.com/70270603/134138316-b0259b51-973a-4568-8c38-5c56ac4d27c5.png)
![15ff1abbfab3c3c313fce7b8ef013d9](https://user-images.githubusercontent.com/70270603/134138339-1a8ff804-c03f-456e-8fa5-fd2bce43ef54.png)

**Comment by qinmi-sudo** [comment]
I have solved it !

**Comment by AnguseZhang** [comment, marked as answer]
See https://github.com/deepmodeling/dpgen/issues/365#issuecomment-834429036
